deps/common/crc16.c

```diff
@@ -63,11 +63,11 @@ static const unsigned short crc16tab[256]= {
 	0xef1f,0xff3e,0xcf5d,0xdf7c,0xaf9b,0xbfba,0x8fd9,0x9ff8,
 	0x6e17,0x7e36,0x4e55,0x5e74,0x2e93,0x3eb2,0x0ed1,0x1ef0
 };
-  
-unsigned short crc16_ccitt(const char *buf, int len)
+
+unsigned short crc16_ccitt(unsigned short crc_start, const char *buf, int len)
```

Comment:
this change is not necessary anymore, let's revert it

Reply:
are you sure?  I think its more useful this way, even if not needed for our use case.  (also requries changing all the callers)

Reply:
It is just to avoid a commit to open source code. Don't know if we should care or not but my instincts say that if we don't need, we shouldn't touch it :) Probably, we should discuss it.